loginstyles.css

```diff
@@ -1,92 +1,78 @@
 /* Navbar Design */
+:root{
+    --heart-icon-color: #f10000;
+    --background-color: #ffff;
+    --light-white: #e9e9e9;
+    --font-dark-black-color: #111;
+    --grey-color-font: #888;
+    --button-color: #0031D1;
+}
 body {
```

Comment:
You can add CSS RESET here

Reply:
SureThanks for the suggestion @irajbhinde